tsfile/src/main/java/org/apache/iotdb/tsfile/utils/ReadWriteIOUtils.java

```diff
@@ -691,6 +692,22 @@ public static Map<String, String> readMap(ByteBuffer buffer) {
     return map;
   }
 
+  public static LinkedHashMap<String, String> readLinkedHashMap(ByteBuffer buffer) {
+    int length = readInt(buffer);
+    if (length == NO_BYTE_TO_READ) {
+      return null;
```

Comment:
It's not a good practice to return null here. It may cause NPE.

I knew that there are a lot of `return null` in this file. I think it's time to do refactor.

Reply:
<img width="531" alt="image" src="https://user-images.githubusercontent.com/30497621/183075614-c1378d29-b3f6-4f5d-adee-444555941bf8.png">

This is not good :(